airgun/views/contentview.py

```diff
@@ -132,7 +132,7 @@ class details(SatTab):
     @View.nested
     class versions(SatTab):
         searchbox = Search()
-        table = SatTable(
+        table = Table(
             locator='//table',
             column_widgets={
```

Comment:
any issues with './/table' ?

Reply:
Maybe, no issues, but that was not intention of current PR. I can look on that in next improvements

Reply:
Ok NP